mqtt/src/test/java/akka/stream/alpakka/mqtt/javadsl/MqttSourceTest.java

```diff
@@ -45,12 +47,19 @@
 
   @BeforeClass
   public static void setup() throws Exception {
+    serverPort = AvailablePort$.MODULE$.nextPort();
+    serverWebSocketPort = AvailablePort$.MODULE$.nextPort();
+
     final Pair<ActorSystem, Materializer> sysmat = setupMaterializer();
     system = sysmat.first();
     materializer = sysmat.second();
 
+    Properties serverProperties = new Properties();
+    serverProperties.setProperty(BrokerConstants.PORT_PROPERTY_NAME, String.valueOf(serverPort));
+    serverProperties.setProperty(BrokerConstants.WEB_SOCKET_PORT_PROPERTY_NAME, String.valueOf(serverWebSocketPort));
+
```

Comment:
According the Travis CI test failures we will have to make `BrokerConstants.PERSISTENT_STORE_PROPERTY_NAME` unique as well.